ovs/v1/ovs.yaml

```diff
@@ -223,13 +160,126 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/transportCall'
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/transportCall'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/error'
+    delete:
+      tags:
+        - Operational Vessel Schedules
+      summary: Delete a transport-call by its transportCallId
+      parameters:
+        - $ref: '#/components/parameters/transportCallID'
+      responses:
+        '204':
+          description: Subscription stopped
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/error'
```

Comment:
Is this a special error - or should global error object be used?

Reply:
Good point, I will discuss this, let us use the global one for now